models/members.js

```diff
@@ -17,7 +17,23 @@ var app = app || {};
 
   Member.fetchMember = (lName) => $.getJSON(_API_URL_ + '/api/v0/members/' + lName).then(data => console.log(data)).catch(err => console.error(errorCallback(err)));
 
-  Member.addMember = (member) => $.post(_API_URL_ + '/api/v0/members/add', member).catch(err => console.error(errorCallback(err)));
+  Member.addMember = (member) => $.ajax(
+    {
+      "async": true,
+      "crossDomain": true,
+      "url": _API_URL_ + "/api/v0/members/add",
+      "method": "POST",
+      "headers": {
+        "Content-Type": "application/json",
+        "Cache-Control": "no-cache",
+        "Postman-Token": "6f60d89d-34a3-8bfb-c2a6-9b4a9c6990b9"
+      },
+      "processData": false,
+      "data": JSON.stringify(member)
+    }
+  ).done(function (response) {
+    console.log(response);
+  });
 
   Member.getSkill = (skill) => $.getJSON(_API_URL_ + '/api/v0/members/' + skill).then(data => console.log(data)).catch(err => console.error(errorCallback(err)));
```

Comment:
This will not work. Because the the route is /api/v0/members?skills=<comma separated list>.

As a sanity measure, you need to check for each incoming parameter.  i.e. skill could be integer, array, string or object.  We need to make sure it's a comma separate list. Further this URL should be URL encoded. If it's not ASCII encoded then skills like 'c#' will look like 'c' at the server end. You're using promises here to await processing the ajax but you don't have an error handler. 

$.ajax().done().error();

Reply:
I haven't gotten to this one yet. Will try to work on this today. I haven't yet worked with an endpath like this or encoded stuff to ascii for api's so it may take me a bit to figure it out. :)

For the addMembers, I tried adding an error key inside the ajax request. I couldn't get the.error() to work.